### PR TITLE
fixed spacing and tasting notes display

### PR DESCRIPTION
**Pull request by billyfung**
Fixed spacing stuff and made everything prettier for you senpai

### CODE REVIEW

**Comment by billyfung** [comment]
screen shot 2016-01-19 at 7 18 32 pm

File: templates/index.html [hunk 1]
```diff
@@ -56,12 +60,16 @@
         {{ coffee.description }}
       </div>
       <div class="extra">
-        <div class="ui label">{{ coffee.notes }}</div>
+      {% for note in coffee.notes.replace('&', ',').split(',') %}
```
Comment (jinpark) [inline templates/index.html]: i wonder if we can do this (the replace and split) in the backend instead. I dont like doing this stuff on the front end

Comment (billyfung) [inline templates/index.html]: Yeah you can

Comment (jinpark) [inline templates/index.html]: on the template*

**Comment by jinpark** [comment]
lgtm

**Pull request merged**